bi/pkg/specs/wait.go

```diff
@@ -44,7 +50,21 @@ func (spec *InstallSpec) WaitForBootstrap(ctx context.Context, kubeClient kube.K
 		l.Info("Finished waiting")
 	}
 
-	return nil
+	// get the access-info configmap (and URL information)
+	info, err := kubeClient.GetAccessInfo(ctx, ns)
+	if err != nil {
+		slog.Error("Failed to get access info config map", slog.Any("err", err))
+		return err
+	}
+
+	// try to make sure the control server is up before we return
+	slog.Debug("Attempting to connect to control server", slog.String("url", info.GetURL()))
+	err = retry.Do(func() error {
+		_, err := http.Head(info.GetURL())
+		return err
+	}, retry.Context(ctx))
```

Comment:
This makes sure that we can actually connect to the control server after we've checked that the deployment is "finished" rolling out.